opensource/project/software-req-win.md

```diff
@@ -34,229 +39,41 @@ fine. All the Docker project repositories are public and visible to everyone.
-
-3. Run the installer.
-
-	The system opens the **Go Programming Language Setup** dialog.
-
-4. Select all the defaults to install.
-
-5. Press **Finish** to close the installation dialog.
-
-6. Start a command prompt.
-
-7. Change to the Go `src` directory.
-
-		cd c:\Go\src
-
-8. Set the following Go variables
-
-		c:\Go\src> set GOOS=linux
-		c:\Go\src> set GOARCH=amd64
-
-9. Compile the source.
-
-		c:\Go\src> make.bat
-
-	Compiling the source also adds a number of variables to your Windows environment.
-
-## Task 6. Get the Docker repository
-
-In this step, you start a Git `bash` terminal and get the Docker source code
-from GitHub.
-
-1. Locate the **Git Bash** program and start it.
-
-	Recall that **Git Bash** came with the Git for Windows installation.  **Git
-	Bash** just as it sounds allows you to run a Bash terminal on Windows.
-
-	![Git Bash](images/git_bash.png)
-
-2. Change to the root directory.
-
-		$ cd /c/
-
-3. Make a `gopath` directory.
-
-		$ mkdir gopath
-
-4. Go get the `docker/docker` repository.
-
-		$ go.exe get github.com/docker/docker package github.com/docker/docker
-        imports github.com/docker/docker
-        imports github.com/docker/docker: no buildable Go source files in C:\gopath\src\github.com\docker\docker
-
-	In the next steps, you create environment variables for you Go paths.
-
-5. Open the **Control Panel** on your system.
-
-6. Choose **System and Security > System**.
-
-7. Click the **Advanced system settings** link in the sidebar.
-
-	The system opens the **System Properties** dialog.
+## 3. Install Git for Windows
 
-8. Select the **Advanced** tab.
+In PowerShell, run:
 
-9. Click **Environment Variables**.
+    Invoke-Webrequest "https://github.com/git-for-windows/git/releases/download/v2.7.2.windows.1/Git-2.7.2-64-bit.exe" -OutFile git.exe -UseBasicParsing
+    Start-Process git.exe -ArgumentList '/VERYSILENT /SUPPRESSMSGBOXES /CLOSEAPPLICATIONS /DIR=c:\git\' -Wait
+    setx /M PATH "$env:Path;c:\git\cmd"
 
-	The system opens the **Environment Variables dialog** dialog.
+## 4. Clone the Docker repository
 
-10. Locate the **System variables** area and scroll to the **Path**
-   variable.
+In a new (to pick up the path change) PowerShell prompt, run:
 
-11. Click **New**.
+    git clone https://github.com/docker/docker
+    cd docker
 
-	Now you are going to create some new variables. These paths you'll create in the next procedure; but you can set them now.
+## 5. Build and run Docker
 
-12. Enter `GOPATH` for the **Variable Name**.
+Create a builder-container with the Docker source code. You can change the source code on your system and rebuild at your leisure:
 
-13. For the **Variable Value** enter the following:
+    docker  build -t docker-builder -f .\Dockerfile.windows .
 
-		C:\gopath;C:\gopath\src\github.com\docker\docker\vendor
+To build Docker, run:
 
+    docker run --name out docker-builder sh -c 'cd /c/go/src/github.com/docker/docker; hack/make.sh binary'
```

Comment:
Should we use a bind-mount (like we use with `BIND_DIR=.` on Linux?) that way people can make code changes on their host, and rebuild in the container, also it saves them from copying out the binary (as is explained in the next step)